### PR TITLE
[NEW] Introduce CallInfo component in contextual-bar for VoIP

### PR DESCRIPTION
**Pull request by tiagoevanp**

### CODE REVIEW

File: client/views/omnichannel/directory/chats/contextualBar/VoipInfo/index.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+import { useRoom } from '../../../../../room/contexts/RoomContext';
+import { VoipInfo } from './VoipInfo';
+
+const VoipInfoWithData = ({ tabBar: { close } }: any): ReactElement => {
+	const room = useRoom();
+	console.log(room);
+	const { servedBy } = room;
+
+	const guest = {};
```
Comment (MartinSchoeler) [inline client/views/omnichannel/directory/chats/contextualBar/VoipInfo/index.tsx:11]: I think all the data you need for the guest can be acquired from an IVoipRoom I did this on a pretty similar component 

`	const { servedBy, queue, v, fname, callDuration, callTotalHoldTime, callEndedAt, callWaitingTime } = room; `

File: client/views/omnichannel/directory/chats/contextualBar/VoipInfo/VoipInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import { useTranslation } from '../../../../../../contexts/TranslationContext';
+import AgentField from '../AgentField';
+import { InfoField } from './InfoField';
+
+type VoipInfoPropsType = {
+	guest: any;
+	servedBy: IRoom['servedBy'];
+	onClickClose: () => void;
+	onClickReport: () => void;
+	onClickCall: () => void;
```
Comment (MartinSchoeler) [inline client/views/omnichannel/directory/chats/contextualBar/VoipInfo/VoipInfo.tsx:15]:
```suggestion
	room: IVoipRoom;
	onClickClose: () => void;
	onClickReport: () => void;
	onClickCall: () => void;
```

File: client/views/omnichannel/directory/chats/contextualBar/VoipInfo/VoipInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import { Box, Button, ButtonGroup, Icon } from '@rocket.chat/fuselage';
+import React, { ReactElement } from 'react';
+
+import { IRoom } from '../../../../../../../definition/IRoom';
```
Comment (MartinSchoeler) [inline client/views/omnichannel/directory/chats/contextualBar/VoipInfo/VoipInfo.tsx:4]:
```suggestion
import { IVoipRoom } from '../../../../../../../definition/IRoom';
```

File: client/views/omnichannel/directory/chats/contextualBar/VoipInfo/VoipInfo.tsx (hunk 2 above)
Comment (MartinSchoeler) [reply client/views/omnichannel/directory/chats/contextualBar/VoipInfo/VoipInfo.tsx:15]: And so on...

File: client/views/omnichannel/directory/chats/contextualBar/VoipInfo/index.tsx (hunk 1 above)
Comment (tiagoevanp) [reply client/views/omnichannel/directory/chats/contextualBar/VoipInfo/index.tsx:11]: For sure, this is not ready... I did the fix earlier... 🤣

**Review by MartinSchoeler — COMMENTED** [review]
Added some pointers to help you out with the typing

File: client/views/omnichannel/directory/chats/contextualBar/VoipInfo/index.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import React, { ReactElement } from 'react';
+
+import { useVoipRoom } from '../../../../../room/contexts/RoomContext';
+import { VoipInfo } from './VoipInfo';
+
+const VoipInfoWithData = ({ tabBar: { close } }: any): ReactElement => {
+	const room = useVoipRoom();
+	console.log(room);
```
Comment (MartinSchoeler) [inline client/views/omnichannel/directory/chats/contextualBar/VoipInfo/index.tsx:8]: Console log here
```suggestion
```

File: app/livechat/server/api/v1/visitor.ts [hunk 5]
```diff
@@ -38,6 +38,7 @@ API.v1.addRoute('livechat/visitor', {
 		}
 
 		guest.connectionData = normalizeHttpHeaderData(this.request.headers);
+
```
Comment (MartinSchoeler) [inline app/livechat/server/api/v1/visitor.ts:41]:
```suggestion
```

**Review by MartinSchoeler — CHANGES_REQUESTED** [review]

File: client/views/omnichannel/directory/chats/contextualBar/VoipInfo/index.tsx (hunk 4 above)
Comment (tiagoevanp) [reply client/views/omnichannel/directory/chats/contextualBar/VoipInfo/index.tsx:8]: fixed

**Review by MartinSchoeler — APPROVED** [review]

**Pull request merged**